lnwallet/channel.go

```diff
@@ -3489,6 +3474,23 @@ func (lc *LightningChannel) validateCommitmentSanity(theirLogCounter,
 	return nil
 }
 
+// ErrZeroHtlc returns a structured error for zero value htlcs.
+func ErrZeroHtlc() *lnwire.CodedError {
```

Comment:
Realized that these aren't actually surfaced to peer as is, will probably drop this commit?